modules/configuring-hybrid-ovnkubernetes.adoc

```diff
@@ -83,3 +99,10 @@ status: {}
 <4> Specify the CIDR configuration used for nodes on the additional overlay network. The `hybridClusterNetwork` CIDR cannot overlap with the `clusterNetwork` CIDR.
 
 . Optional: Back up the `<installation_directory>/manifests/cluster-network-03-config.yml` file. The installation program deletes the `manifests/` directory when creating the cluster.
+
+. You may now proceed with the installation:
++
+[source,terminal]
+----
+$ ./openshift-install create cluster --dir=<installation_directory>
+----
```

Comment:
See first comment; since this is used in the installation section for network customizations, this is covered later. I do agree we can give some directive to continue the installation process, but it would be more of a general note than a specific step.

Reply:
+1 a general note is all that's needed.